css/rheviu.css

```diff
@@ -3,6 +3,10 @@ body {
     overflow-x: hidden;
 }
 
+.logo {
```

Comment:
Instead of implementing a new class, we could use `rounded` or `rounded-sm` to achieve the same effect.

Reply:
Done